docs/guide.rst

```diff
@@ -398,6 +398,41 @@ Sometimes you forget something, or just things may go wrong and
 you need another try. In such case add ``-f`` or ``--force`` to
 quickly overwrite existing files with the right content.
 
+.. _linking_tmt_objects:
+
+Linking tmt objects
+------------------------------------------------------------------
+If you create a Jira issue for a bug or a feature, you can link it
+to the test or plan that covers it. This can be either done automatically
+on creation of a new test or plan, or later on using the ``tmt link`` command.
+
+.. code-block:: shell
+
+    tmt link verifies:issues.redhat.com/browse/YOUR-ISSUE tests/core/smoke
+
+For this feature to be enabled, you have to create a configuration node in a
+configuration tree. Once the configuration is present, it automatically enables
+the linking on its own, no further actions needed.
+The configuration has a reserved keywork ``linking`` and should
+have following structure:
+
+.. code-block:: yaml
+
+    linking:
+      - type: jira
+        server: https://issues.redhat.com
+        service: http://localhost:8000/
+        token: <YOUR_PERSONAL_JIRA_TOKEN>
+
+The ``type`` key specifies the type of the issue tracking service
+you want to link to (so far only Jira is supported).
+The ``server`` is the URL of said service. The ``service`` is the URL of the
+service that presents a tmt metadata in a human-readable form.
```

Comment:
The difference between `server` and `service` is fairly minimal and got me thinking about which one is which. How about renaming these keys, e.g. `server` => `website`, `jira-website` (together with `jira-token`? Because a mere `token` - authenticate the user, sure, but against which service, the linking one or Jira?), `service` => something like `tmt-link-service` or something making it clear which service it is. At first, I thought it referred to Jira API...

Reply:
hmm, I wouldn't go with `web`. `tmt-web` would be better imho, or anything that says that it's the service that will do the linking - proxy, api server, ...

Reply:
Summary from the discussion: `tmt-web-url`